ballerina-tests/tests/01_listener_test.bal

```diff
@@ -64,37 +54,27 @@ function testAttachingGraphQLServiceWithAnnotationToDynamicListener() returns er
 }
 
 @test:Config {
-    groups: ["listener"]
+    groups: ["listener"],
+    dataProvider: dataProviderListener
 }
-function testAttachServiceWithQueryToHttp2BasedListener() returns error? {
-    string document = string `query { person{ age } }`;
-    string url = "http://localhost:9190/service_with_http2";
+function testListener(string url, string documentFileName) returns error? {
+    string document = check getGraphQLDocumentFromFile(appendGraphqlExtension(documentFileName));
```

Comment:
We don't need a separate function to do this, right? We can append the extension inside the `getGraphQLDocumentFromFile` function. Just noticed, the function name should be `getGraphqlDocumentFromFile`.